README.md

```diff
@@ -38,10 +67,21 @@ If you experience a missing locale error, you can try configuring the locale:
 ```
 python manage.py migrate
 python manage.py createsuperuser
-python manage.py runserver
+python manage.py runserver 8000
```

Comment:
Not sure why this change was needed, isn't localhost:8000 the default? :)

Reply:
oh, yeah, wasn't sure about that, but checked django's course and you are totally right 👍